pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -444,6 +445,24 @@ private Long configureRefreshDelay(final Long refreshDelay) {
   )
   private final Collection<String> accountsWhitelist = null;
 
+  @Option(
+    names = {"--privacy-url"},
+    description = "The URL on which enclave is running "
+  )
+  private final URI privacyUrl = PrivacyParameters.DEFAULT_ORION_URI;
+
+  @Option(
+    names = {"--privacy-public-key"},
```

Comment:
We did not update the CLI guideline yet but to comply with the next CLI refactoring that will be released in the 0.9 version, you could already name this option `--privacy-public-key-file`